error.js

```diff
@@ -4,7 +4,7 @@ class CustomError extends Error {
     super();
     Error.captureStackTrace(this, this.constructor);
     this.name = name || 'JoshError';
-    this.message = message;
+    this.message = `\x1b[31m${message}\x1b[0m`;
```

Comment:
I don't really agree with change. Adding colors seems alright for the console but if we're not sending to the console then this causes issues (say someone returns the message to a browser or whatever).

Reply:
Resolved, removed colors.

Reply:
Can always add a color-console option to the options, but later on.